src/structures/game_session.ts

```diff
@@ -960,6 +960,21 @@ export default class GameSession {
             .onConflict("player_id")
             .ignore();
 
+        await dbContext.kmq("daily_player_stats")
```

Comment:
Instead of having 3 different tables, can't we use the same table, and filter daily/weekly/monthly with a WHERE condition on date?